go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/aws/aws-sdk-go v1.38.49
 	github.com/gofrs/flock v0.7.0
+	github.com/logandavies181/arnlike v1.1.0
```

Comment:
This library is used for quite a critical check, I would suggest re-implementing in `aws-iam-authenticator` rather than depending on a library that's new and not used anywhere else

Reply:
I agree, this library is more security-sensitive, and the amount of code is not that much either https://github.com/logandavies181/arnlike/blob/main/arnlike.go, I think it makes more sense to move that code here, and have the code audited and reviewed within this repo

Reply:
If one of the maintainers requests it, I'll be happy to oblige. However, all the dependencies for this project are vendored, and the whole source of that module is available in this PR, so I'm sure the security folks will be reviewing it

Reply:
@logandavies181 actually one of the comments from the security reviewers was to move this small library into the main Go package instead of importing as a library. Could you do that please? Sorry I missed this on earlier reviews..

Reply:
@nckturner this has been done on Jay's request